newIDE/app/src/BehaviorsEditor/index.js

```diff
@@ -50,6 +50,26 @@ type Props = {|
 |};
 
 const BehaviorsEditor = (props: Props) => {
+  const scrollView = React.useRef<?ScrollViewInterface>(null);
+  const justAddedBehaviorName = React.useRef((null: ?string));
+  const justAddedBeaviorAccordionElement = React.useRef(
```

Comment:
Why use ref instead of state variables here?

Reply:
good question, it felt more intuitive to avoid multiple renders, but maybe you're right I'll try with state